notebooks/official/feature_store/sdk-feature-store-pandas.ipynb

```diff
@@ -30,15 +30,23 @@
       },
```

Comment:
<blockquote>Missing data </blockquote><blockquote><br></blockquote><blockquote>Recall Batch Serve from the last ingestion has some missing data in it.</blockquote><p>I think this needs a bit more context/explanation. We didn't mention this previously (that data was missing). </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='67'/>